libs/employees/src/lib/employees/employees.component.ts

```diff
@@ -1,38 +1,24 @@
 import { Component, OnInit } from '@angular/core';
+//import { Subscription } from 'rxjs';
 import { TableColumn } from '@razroo-fully-architected-dashboard/ui/common';
 
-const ELEMENT_DATA = [
-  {name: 'test', position: 'test position', location: 'New York',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'Los Angelos',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'Alabama',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-];
-
 @Component({
   selector: 'razroo-fully-architected-dashboard-employees',
   templateUrl: './employees.component.html',
-  styleUrls: ['./employees.component.scss']
+  styleUrls: ['./employees.component.scss'],
+ 
 })
+
 export class EmployeesComponent implements OnInit {
-  elementData;
-  employeeTableColumns: TableColumn[];
+  
 
-  constructor() { }
+
+  employeeTableColumns: TableColumn[];
 
   ngOnInit(): void {
-    this.elementData = ELEMENT_DATA;
+
```

Comment:
After added service to the constructor, will then go ahead and call the service within the ngOnInit block(just to get a feel for it). The following is the syntax to do so. 
```
this.employeesService.subscribe(data => {
  console.log('data');
  console.log(data);
});
```

This with the import, constructor, and subscribe within ngOnInit should do the trick